docs/src/componentDocs/GettingStarted/ReactNativeTesting.mdx

```diff
@@ -0,0 +1,146 @@
+
+There are many [varieties of tests](https://www.geeksforgeeks.org/types-software-testing/), each with its own intended purpose. The most common types of tests are:
+
+-   **Unit Tests**: Verify individual methods, functions, or components.
+-   **Integration Tests**: Verify that different modules or services in your application work together.
+-   **Functional Tests**: Verify the output of certain actions based on business requirements.
+-   **End-to-end (E2E) Tests**: Verify that user behaviors / workflows function as expected.
+-   **Acceptance Tests**: Verify that the entire system meets the business requirements.
+
+This document focuses mainly on Unit Tests and E2E tests, as they are the easiest to automate.
+
+<Divider />
+
+# Unit Tests
+
+Unit testing is a method of testing the smallest pieces of your application or the components of the application and is used to ensure that the individual units of the software are working as intended.
+
+Unit tests are usually automated and are designed to test specific parts of the code, such as a particular function or method and should closely resemble how your applications are interacted by the users.
+These will likely make up the bulk of your tests because they are quick to write, easy to automate, and can help you achieve significant code coverage.
+
+> Code Coverage measures the percentage of your code that is verified by your tests.
+
+## When to Write
+
+As long as you are writing tests and achieving acceptable levels of coverage, the _when_ is not as important.
+
+The most important thing to keep in mind about tests is that they should provide confidence that the code and design are working as intended without issues in any scenario. For most teams, it's fine to write tests after features and functions are developed, as long as you write quality tests and achieve good coverage.
+
+## How to Write
+
+When unit testing components, the most effective tests are based on what is rendered on the screen rather than the internal implementation logic. The test should consider the application from the user's perspective and test things the user views and interacts with.
+
+For example, consider testing an Open Button that should open a modal dialog when clicked. Your test should check that a button with the _Open_ label is rendered on the screen, and when it is clicked, the dialog is rendered on the screen with the correct content. You may be tempted to identify the button by a class or id, or test that a particular function is called when the button is pressed - these tests are not as effective because they are more likely to break if the implementation of the component changes, even though the end behavior is the same.
+
+-   Test what the user sees (i.e., rendered output) and what the user can do  (i.e., interactions).
+-   If there is visual change after an interaction, test for it.
+-   If there is a value returned after an action, test for it.
+
+## Testing Frameworks
+
```

Comment:
you can update this section to reference React Native with Jest. Refer to our RN templates for information. Not sure if we need to mention the under-the-hood process here but the npx react-native init myApp uses the rn-community/cli to scaffold the project using our templates....and each template "project" comes with example tests utilizing Jest

Reply:
Added this in react native testing section